### PR TITLE
ONE 10 cards

### PR DESCRIPTION
**Pull request by Simisays**
The end is in sight !

### CODE REVIEW

File: forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Name:Ravenous Necrotitan
+ManaCost:2 B B
+Types:Creature Phyrexian Horror
+PT:6/6
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSacrifice | TriggerDescription$ When CARDNAME enters the battlefield, sacrifice a land. Then conjure two cards named Mishra's Foundry onto the battlefield tapped.
+SVar:TrigSacrifice:DB$ Sacrifice | SacValid$ Creature | Defined$ You | ConditionCheckSVar$ X | ConditionSVarCompare$ LE3
+Corrupted — When CARDNAME enters the battlefield, sacrifice a creature unless an opponent has three or more poison counters.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt:7]: The Description is wrong for the Trigger on line 5 and line 7 should be removed

File: forge-gui/res/cardsfolder/upcoming/reject_imperfection.txt [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Name:Reject Imperfection
+ManaCost:1 U U
+Types:Instant
+A:SP$ Counter | Cost$ 1 U | TargetType$ Spell | ValidTgts$ Card | ConditionCheckSVar$ X | ConditionSVarCompare$ LE3 | SubAbility$ DBProliferate | SpellDescription$ SpellDescription$ Counter target spell. If that spell's mana value was 3 or less, proliferate. (Choose any number of permanents and/or players, then give each another counter of each kind already there.)
+SVar:DBProliferate:DB$ Proliferate
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/reject_imperfection.txt:5]: the condition should be on the Proliferate part

File: forge-gui/res/cardsfolder/upcoming/shrapnel_slinger.txt [hunk 3]
```diff
@@ -0,0 +1,10 @@
+Name:Shrapnel Slinger
+ManaCost:1 R
+Types:Artifact Creature — Phyrexian Beast
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$
+When CARDNAME enters the battlefield, you may sacrifice a creature. When you do, destroy target artifact an opponent controls.
+SVar:TrigSac:AB$ ImmediateTrigger | Cost$ Sac<1/Creature.Other/another creature> | Execute$ TrigDestroy | TriggerDescription$ When you do, destroy target artifact an opponent controls.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/shrapnel_slinger.txt:7]: doesn't say other, just "Creature"

File: forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt [hunk 4]
```diff
@@ -0,0 +1,11 @@
+Name:Ravenous Necrotitan
+ManaCost:2 B B
+Types:Creature Phyrexian Horror
+PT:6/6
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSacrifice | TriggerDescription$ When CARDNAME enters the battlefield, sacrifice a land. Then conjure two cards named Mishra's Foundry onto the battlefield tapped.
+SVar:TrigSacrifice:DB$ Sacrifice | SacValid$ Creature | Defined$ You | ConditionCheckSVar$ X | ConditionSVarCompare$ LE3
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt:6]: LT3

File: forge-gui/res/cardsfolder/upcoming/shrapnel_slinger.txt [hunk 5]
```diff
@@ -0,0 +1,10 @@
+Name:Shrapnel Slinger
+ManaCost:1 R
+Types:Artifact Creature — Phyrexian Beast
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/shrapnel_slinger.txt:3]: No - in types ⚠️

File: forge-gui/res/cardsfolder/upcoming/shrapnel_slinger.txt [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+PT:2/2
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSac | TriggerDescription$
+When CARDNAME enters the battlefield, you may sacrifice a creature. When you do, destroy target artifact an opponent controls.
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/shrapnel_slinger.txt:6]: wrong linebreak

File: forge-gui/res/cardsfolder/upcoming/reject_imperfection.txt [hunk 7]
```diff
@@ -0,0 +1,8 @@
...
+SVar:X:Targeted$CardManaCost
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/reject_imperfection.txt:6]: `SpellTargeted$CardManaCostLKI`

File: forge-gui/res/cardsfolder/upcoming/reject_imperfection.txt [hunk 8]
```diff
@@ -0,0 +1,8 @@
+Name:Reject Imperfection
+ManaCost:1 U U
+Types:Instant
+A:SP$ Counter | Cost$ 1 U | TargetType$ Spell | ValidTgts$ Card | ConditionCheckSVar$ X | ConditionSVarCompare$ LE3 | SubAbility$ DBProliferate | SpellDescription$ SpellDescription$ Counter target spell. If that spell's mana value was 3 or less, proliferate. (Choose any number of permanents and/or players, then give each another counter of each kind already there.)
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/reject_imperfection.txt:4]: no cost for SP needed, it is also wrong

File: forge-gui/res/cardsfolder/upcoming/ruthless_predation.txt [hunk 9]
```diff
@@ -0,0 +1,6 @@
+Name:Ruthless Predation
+ManaCost:1 G
+Types:Sorcery
+A:SP$ Pump | Cost$ 1 G | ValidTgts$ Creature.YouCtrl | AILogic$ Fight | TgtPrompt$ Select target creature you control | SubAbility$ DBFight | NumAtt$ +1 | NumDef$ +2 | SpellDescription$ Target creature you control gets +1/+2 until end of turn. It fights target creature you don't control.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/ruthless_predation.txt:4]: no cost for SP needed

File: forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt [hunk 10]
```diff
@@ -2,9 +2,8 @@ Name:Ravenous Necrotitan
 ManaCost:2 B B
 Types:Creature Phyrexian Horror
 PT:6/6
-T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigSacrifice | TriggerDescription$ When CARDNAME enters the battlefield, sacrifice a land. Then conjure two cards named Mishra's Foundry onto the battlefield tapped.
-SVar:TrigSacrifice:DB$ Sacrifice | SacValid$ Creature | Defined$ You | ConditionCheckSVar$ X | ConditionSVarCompare$ LE3
-Corrupted — When CARDNAME enters the battlefield, sacrifice a creature unless an opponent has three or more poison counters.
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | CheckSVar$ X | SVarCompare$ LT3 | Execute$ TrigSacrifice | TriggerDescription$ Corrupted — When CARDNAME enters the battlefield, sacrifice a creature unless an opponent has three or more poison counters.
```
Comment (Hanmac) [inline forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt:5]: @tool4ever i need to ask Judges, does the ability trigger when Corrupted isn't active

but only is active when the ability resolves?

Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt:5]: so it should not be a trigger condition, but a condition on the spellAbility

Comment (tool4ever) [reply forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt:5]: yea that shouldn't have been changed

File: forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt [hunk 11]
```diff
@@ -0,0 +1,10 @@
+Name:Ravenous Necrotitan
+ManaCost:2 B B
+Types:Creature Phyrexian Horror
+PT:6/6
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | ConditionCheckSVar$ X | ConditionSVarCompare$ LT3 | Execute$ TrigSacrifice | TriggerDescription$ Corrupted — When CARDNAME enters the battlefield, sacrifice a creature unless an opponent has three or more poison counters.
+SVar:TrigSacrifice:DB$ Sacrifice | SacValid$ Creature | Defined$ You
+SVar:X:PlayerCountOpponents$HighestPoisonCounters
+DeckHints$Keyword$Toxic|Infect
```
Comment (tool4ever) [inline forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt:8]: $ = :

File: forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt (hunk 10 above)
Comment (Hanmac) [reply forge-gui/res/cardsfolder/upcoming/ravenous_necrotitan.txt]: @Simisays ^

**Review by tool4ever — APPROVED** [review]
nice work

**Pull request merged**